data/locale/fr-FR.ini

```diff
@@ -30,8 +30,8 @@ NDIPlugin.OutputSettings.Main.Name="Nom NDI™"
 NDIPlugin.OutputSettings.Preview.Name="Nom NDI™"
 NDIPlugin.FilterName="Sortie NDI™ dédiée"
 NDIPlugin.AudioFilterName="Sortie NDI™ dédiée (Audio seul)"
-NDIPlugin.PremultipliedAlphaFilterName="obs-ndi - Correction du compositing alpha"
-NDIPlugin.LibError.Title="Runtime NDI™ introuvable"
-NDIPlugin.LibError.Message.Win="La bibliothèque NDI™ est introuvable sur votre système.<br>Téléchargez l'installateur du runtime ici : <a href='http://new.tk/NDIRedistV5'>http://new.tk/NDIRedistV5</a>"
-NDIPlugin.LibError.Message.macOS="Le bibliothèque NDI™ est introuvable sur votre système.<br>Téléchargez l'installateur du runtime ici : <a href='http://new.tk/NDIRedistV5Apple'>http://new.tk/NDIRedistV5Apple</a>"
-NDIPlugin.LibError.Message.Linux="La bibliothèque NDI™ est introuvable. Installez libndi sur votre système.<br>Vous pouvez en trouvez une copie sur la page GitHub d'obs-ndi."
+NDIPlugin.PremultipliedAlphaFilterName="obs-ndi - Correction de simulation de transparence"
+NDIPlugin.LibError.Title="Bibliothèque NDI™ introuvable"
```

Comment:
Google Translate says English to French of `NDI™ Runtime not found` is `Exécution NDI™ introuvable`.
Reverse translation gives back `NDI™ runtime not found`.
 
`Bibliothèque NDI™ introuvable` translates back as `NDI™ library not found`.

Sure, I can forgive calling it a library, but `runtime` sees more precise.

So, which seems better:
1. `Exécution NDI™ introuvable`
-or-
2. `Bibliothèque NDI™ introuvable`